packages/react/core/README.md

```diff
@@ -41,3 +41,95 @@ function App() {
+
+### `useDisconnect(uiWallet)`
+
+Given a `UiWallet`, this hook returns a function you can call to ask the wallet to deauthorize accounts authorized for the current domain, or at least to remove them from the wallet's `accounts` array for the time being, depending on the wallet's implementation of `standard:disconnect`. Call the returned method will return a promise that resolves once the disconnection is complete.
+
+```tsx
+function DisconnectButton({ onDisconnected, wallet }) {
+    const [isDisconnecting, disconnect] = useDisconnect(wallet);
+    return (
+        <button
+            disabled={isDisconnecting}
+            onClick={() => {
+                disconnect.then(onDisconnected);
+            }}
+        >
+            Disconnect
+        </button>
+    );
+}
+```
+
+### `useSignMessage(uiWalletAccount)`
+
+Given a `UiWalletAccount`, this hook returns a function you can call to sign a byte array using that account's private key.
+
+```tsx
+function MessageSignerPanel({ uiWalletAccount }) {
+    const signMessage = useSignMessage(uiWalletAccount);
+    const textareaRef = createRef();
+    const handleClick = useCallback(() => {
+        const message = textareaRef.current.value;
+        const messageBytes = new TextEncoder().encode(message);
+        signMessage(messageBytes).then(({ signature, signedMessage }) => {
+            console.log('You signed a message!', message, new TextDecoder('utf8').decode(signedMessage), signature);
+        });
+    }, [signMessage]);
+    return (
+        <section>
+            <textarea ref={textareaRef}></textarea>
+            <button onClick={handleClick}>Sign Message</button>
+        </section>
+    );
+}
+```
+
+### `useSignAndSendTransaction(uiWalletAccount, chain)`
+
+Given a `UiWalletAccount` and the `IdentifierString` representation of a chain, this hook returns a function you can call to sign and send a byte-encoded transaction.
+
+```tsx
+function DemoTransaction({ chain, uiWalletAccount }) {
+    const signAndSendTransaction = useSignAndSendTransaction(uiWalletAccount, chain);
+    const quantityRef = createRef();
+    const handleClick = useCallback(() => {
+        const quantity = parseInt(quantityRef.current.value, 10);
+        const transactionBytes = new Uint8Array(/* exercise left to the reader */);
+        signAndSendTransaction(transactionBytes).then((signature) => {
+            console.log('Transaction sent!', signature);
+        });
+    }, [signAndSendTransaction]);
+    return (
+        <section>
+            <label>
+                Quantity: <input type="number" ref={quantityRef} />
+            </label>
+            <button onClick={handleClick}>Purchase</button>
+        </section>
+    );
+}
+```
```

Comment:
These are not `standard:` features, they are all chain-specific and shouldn't be generalized.

Reply:
Yeah, these are backed by `experimental:sign*` features. Your suggestion is just to delete these, right? No React hooks for the `experimental:` features?